test/Shared/UserManagerTestBase.cs

```diff
@@ -1050,41 +1051,43 @@ public Task<IdentityResult> ValidateAsync(UserManager<TUser> manager, TUser user
             roleClaims = await manager.GetClaimsAsync(role);
             Assert.Equal(0, roleClaims.Count);
 
-            IdentityResultAssert.VerifyRoleManagerSuccessLog(manager.Logger, "RemoveClaimAsync", role.Id.ToString());
+            IdentityResultAssert.VerifyRoleManagerSuccessLog(manager.Logger, "RemoveClaimAsync", await manager.GetRoleIdAsync(role));
         }
 
         [Fact]
         public async Task CanRoleFindById()
         {
             var manager = CreateRoleManager();
-            var role = CreateRole("FindByIdAsync");
-            Assert.Null(await manager.FindByIdAsync(role.Id.ToString()));
+            var role = CreateTestRole("FindByIdAsync");
```

Comment:
we do add randomization in the helper when there is no argument provide. When something is, then we consider it to be fixed since they might need to compare that value later. I agree though that I missed it in this method